style/style.css

```diff
@@ -320,3 +333,210 @@ textarea {
 .btn-contact {
   width: 48%;
 }
+
+/*
+      ==============================================
+       *** MEDIA QUERY - Desktop Version- 1024px ***
+      ==============================================
+*/
+
+@media only screen and (min-width: 1024px) {
+  /* ------- Main ------- */
+  main {
+    background-image: url("../images/header-shapes-desktop.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: center;
+    padding: 288px 24px 255px 24px;
```

Comment:
- [ ] You are adding a padding-top that is too big and makes your website a little bit weird.

Your design:

![image](https://user-images.githubusercontent.com/60631456/121529457-09315b00-c9c2-11eb-8c79-bc3dcc352b0a.png)

Original design:

![image](https://user-images.githubusercontent.com/60631456/121529539-1cdcc180-c9c2-11eb-8026-a91f3a9a6687.png)

Reply:
Solved